db/migrate/20161107112025_remove_available_currencies_column_from_communities.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAvailableCurrenciesColumnFromCommunities < ActiveRecord::Migration
+  def up
+    remove_column :communities, :available_currencies
+  end
```

Comment:
You can (and probably should) add a down migration to `CopyCommunityCurrencyToCurrencyColumn`, where `SET c.available_currency = c.currency`